src/basics.md

```diff
@@ -142,10 +163,35 @@ use rand::Rng;
 
 fn main() {
     let mut rng = rand::thread_rng();
-    println!("{}", rng.gen_range(0, 10));
+    println!("Integer: {}", rng.gen_range(0, 10));
+    println!("Float: {}", rng.gen_range(0.0, 10.0));
 }
 ```
 
+Alternatively, one can use the [`Range`](https://doc.rust-lang.org/rand/rand/distributions/range/struct.Range.html) distribution.
```

Comment:
on the other hand "simulating a six sided die roll" might be a problem that merits its own example so we might decide to either extract it as a separate one or rewrite one of the previously mentioned ones 👍